src/validation/trust.information.validation.ts

```diff
@@ -1,7 +1,43 @@
 import { body } from "express-validator";
 
 import { ErrorMessages } from "./error.messages";
+import { trustType } from "../model";
+import { TrustKey } from "../model/trust.model";
 
 export const trustInformation = [
   body("trusts").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.TRUST_DATA_EMPTY)
 ];
+
+export type TrustValidationError = {
+  param: any;
+  msg: any;
+};
+
+export const checkMandatoryTrustFields = (trusts: trustType.Trust[]): TrustValidationError[] => {
+  const errors: TrustValidationError[] = [];
+  trusts.forEach(trust => {
```

Comment:
forEach won't wait for each item to be processed so may return errors as empty before each trust => {.. } function has a chance to run. 
Mo has recommended to me before to use the .map function when iterating over an array, or failing that use the for(item of items)

Reply:
So this comment applies to change on line 127 of `trust.information.controller.ts` too @markpit ?

Reply:
yes it'll apply to 127 as well